[28/Nov/2014 11:50:17] ERROR [django.request:231] Internal Server Error: /processoren/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 98, in get_response
    resolver_match = resolver.resolve(request.path_info)
  File "/usr/local/lib/python2.7/dist-packages/django/core/urlresolvers.py", line 340, in resolve
    sub_match = pattern.resolve(new_path)
  File "/usr/local/lib/python2.7/dist-packages/django/core/urlresolvers.py", line 224, in resolve
    return ResolverMatch(self.callback, args, kwargs, self.name)
  File "/usr/local/lib/python2.7/dist-packages/django/core/urlresolvers.py", line 231, in callback
    self._callback = get_callable(self._callback_str)
  File "/usr/local/lib/python2.7/dist-packages/django/utils/lru_cache.py", line 101, in wrapper
    result = user_function(*args, **kwds)
  File "/usr/local/lib/python2.7/dist-packages/django/core/urlresolvers.py", line 97, in get_callable
    mod = import_module(mod_name)
  File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/home/rik/A-Pc/Website/pcbuilder/views.py", line 121
SyntaxError: Non-ASCII character '\xe2' in file /home/rik/A-Pc/Website/pcbuilder/views.py on line 121, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details
[28/Nov/2014 11:50:17] ERROR [django.request:231] Internal Server Error: /processoren/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 98, in get_response
    resolver_match = resolver.resolve(request.path_info)
  File "/usr/local/lib/python2.7/dist-packages/django/core/urlresolvers.py", line 340, in resolve
    sub_match = pattern.resolve(new_path)
  File "/usr/local/lib/python2.7/dist-packages/django/core/urlresolvers.py", line 224, in resolve
    return ResolverMatch(self.callback, args, kwargs, self.name)
  File "/usr/local/lib/python2.7/dist-packages/django/core/urlresolvers.py", line 231, in callback
    self._callback = get_callable(self._callback_str)
  File "/usr/local/lib/python2.7/dist-packages/django/utils/lru_cache.py", line 101, in wrapper
    result = user_function(*args, **kwds)
  File "/usr/local/lib/python2.7/dist-packages/django/core/urlresolvers.py", line 97, in get_callable
    mod = import_module(mod_name)
  File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/home/rik/A-Pc/Website/pcbuilder/views.py", line 121
SyntaxError: Non-ASCII character '\xe2' in file /home/rik/A-Pc/Website/pcbuilder/views.py on line 121, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details
[28/Nov/2014 11:50:17] ERROR [django.request:231] Internal Server Error: /processoren/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 98, in get_response
    resolver_match = resolver.resolve(request.path_info)
  File "/usr/local/lib/python2.7/dist-packages/django/core/urlresolvers.py", line 340, in resolve
    sub_match = pattern.resolve(new_path)
  File "/usr/local/lib/python2.7/dist-packages/django/core/urlresolvers.py", line 224, in resolve
    return ResolverMatch(self.callback, args, kwargs, self.name)
  File "/usr/local/lib/python2.7/dist-packages/django/core/urlresolvers.py", line 231, in callback
    self._callback = get_callable(self._callback_str)
  File "/usr/local/lib/python2.7/dist-packages/django/utils/lru_cache.py", line 101, in wrapper
    result = user_function(*args, **kwds)
  File "/usr/local/lib/python2.7/dist-packages/django/core/urlresolvers.py", line 97, in get_callable
    mod = import_module(mod_name)
  File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/home/rik/A-Pc/Website/pcbuilder/views.py", line 121
SyntaxError: Non-ASCII character '\xe2' in file /home/rik/A-Pc/Website/pcbuilder/views.py on line 121, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details
[28/Nov/2014 11:50:18] ERROR [django.request:231] Internal Server Error: /processoren/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 98, in get_response
    resolver_match = resolver.resolve(request.path_info)
  File "/usr/local/lib/python2.7/dist-packages/django/core/urlresolvers.py", line 340, in resolve
    sub_match = pattern.resolve(new_path)
  File "/usr/local/lib/python2.7/dist-packages/django/core/urlresolvers.py", line 224, in resolve
    return ResolverMatch(self.callback, args, kwargs, self.name)
  File "/usr/local/lib/python2.7/dist-packages/django/core/urlresolvers.py", line 231, in callback
    self._callback = get_callable(self._callback_str)
  File "/usr/local/lib/python2.7/dist-packages/django/utils/lru_cache.py", line 101, in wrapper
    result = user_function(*args, **kwds)
  File "/usr/local/lib/python2.7/dist-packages/django/core/urlresolvers.py", line 97, in get_callable
    mod = import_module(mod_name)
  File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/home/rik/A-Pc/Website/pcbuilder/views.py", line 121
SyntaxError: Non-ASCII character '\xe2' in file /home/rik/A-Pc/Website/pcbuilder/views.py on line 121, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details
[28/Nov/2014 11:50:48] ERROR [django.request:231] Internal Server Error: /processoren/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/rik/A-Pc/Website/pcbuilder/views.py", line 122, in processoren
    if float(diestringnaam) < float(minPriceSliderValue):
UnicodeEncodeError: 'decimal' codec can't encode character u'\u20ac' in position 0: invalid decimal Unicode string
[28/Nov/2014 11:50:50] ERROR [django.request:231] Internal Server Error: /processoren/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/rik/A-Pc/Website/pcbuilder/views.py", line 122, in processoren
    if float(diestringnaam) < float(minPriceSliderValue):
UnicodeEncodeError: 'decimal' codec can't encode character u'\u20ac' in position 0: invalid decimal Unicode string
[28/Nov/2014 11:51:15] ERROR [django.request:231] Internal Server Error: /processoren/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/rik/A-Pc/Website/pcbuilder/views.py", line 122, in processoren
    if float(diestringnaam) < float(minPriceSliderValue):
UnicodeEncodeError: 'decimal' codec can't encode character u'\u20ac' in position 0: invalid decimal Unicode string
[28/Nov/2014 11:51:15] ERROR [django.request:231] Internal Server Error: /processoren/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/rik/A-Pc/Website/pcbuilder/views.py", line 122, in processoren
    if float(diestringnaam) < float(minPriceSliderValue):
UnicodeEncodeError: 'decimal' codec can't encode character u'\u20ac' in position 0: invalid decimal Unicode string
[28/Nov/2014 11:51:22] ERROR [django.request:231] Internal Server Error: /processoren/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/rik/A-Pc/Website/pcbuilder/views.py", line 122, in processoren
    if float(diestringnaam) < float(minPriceSliderValue):
UnicodeEncodeError: 'decimal' codec can't encode character u'\u20ac' in position 0: invalid decimal Unicode string
[28/Nov/2014 11:51:41] ERROR [django.request:231] Internal Server Error: /processoren/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/rik/A-Pc/Website/pcbuilder/views.py", line 122, in processoren
    if float(diestringnaam) < float(minPriceSliderValue):
UnicodeEncodeError: 'decimal' codec can't encode character u'\u20ac' in position 0: invalid decimal Unicode string
[28/Nov/2014 11:51:42] ERROR [django.request:231] Internal Server Error: /processoren/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/rik/A-Pc/Website/pcbuilder/views.py", line 122, in processoren
    if float(diestringnaam) < float(minPriceSliderValue):
UnicodeEncodeError: 'decimal' codec can't encode character u'\u20ac' in position 0: invalid decimal Unicode string
[28/Nov/2014 11:51:42] ERROR [django.request:231] Internal Server Error: /processoren/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/rik/A-Pc/Website/pcbuilder/views.py", line 122, in processoren
    if float(diestringnaam) < float(minPriceSliderValue):
UnicodeEncodeError: 'decimal' codec can't encode character u'\u20ac' in position 0: invalid decimal Unicode string
[28/Nov/2014 11:51:42] ERROR [django.request:231] Internal Server Error: /processoren/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/rik/A-Pc/Website/pcbuilder/views.py", line 122, in processoren
    if float(diestringnaam) < float(minPriceSliderValue):
UnicodeEncodeError: 'decimal' codec can't encode character u'\u20ac' in position 0: invalid decimal Unicode string
[28/Nov/2014 11:51:43] ERROR [django.request:231] Internal Server Error: /processoren/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/rik/A-Pc/Website/pcbuilder/views.py", line 122, in processoren
    if float(diestringnaam) < float(minPriceSliderValue):
UnicodeEncodeError: 'decimal' codec can't encode character u'\u20ac' in position 0: invalid decimal Unicode string
[28/Nov/2014 11:51:43] ERROR [django.request:231] Internal Server Error: /processoren/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/rik/A-Pc/Website/pcbuilder/views.py", line 122, in processoren
    if float(diestringnaam) < float(minPriceSliderValue):
UnicodeEncodeError: 'decimal' codec can't encode character u'\u20ac' in position 0: invalid decimal Unicode string
[28/Nov/2014 11:51:43] ERROR [django.request:231] Internal Server Error: /processoren/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/rik/A-Pc/Website/pcbuilder/views.py", line 122, in processoren
    if float(diestringnaam) < float(minPriceSliderValue):
UnicodeEncodeError: 'decimal' codec can't encode character u'\u20ac' in position 0: invalid decimal Unicode string
[28/Nov/2014 11:51:50] ERROR [django.request:231] Internal Server Error: /processoren/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/rik/A-Pc/Website/pcbuilder/views.py", line 122, in processoren
    if float(diestringnaam) < float(minPriceSliderValue):
UnicodeEncodeError: 'decimal' codec can't encode character u'\u20ac' in position 0: invalid decimal Unicode string
[28/Nov/2014 11:51:54] ERROR [django.request:231] Internal Server Error: /processoren/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/rik/A-Pc/Website/pcbuilder/views.py", line 122, in processoren
    if float(diestringnaam) < float(minPriceSliderValue):
UnicodeEncodeError: 'decimal' codec can't encode character u'\u20ac' in position 0: invalid decimal Unicode string
[28/Nov/2014 11:51:54] ERROR [django.request:231] Internal Server Error: /processoren/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/rik/A-Pc/Website/pcbuilder/views.py", line 122, in processoren
    if float(diestringnaam) < float(minPriceSliderValue):
UnicodeEncodeError: 'decimal' codec can't encode character u'\u20ac' in position 0: invalid decimal Unicode string
[28/Nov/2014 11:51:55] ERROR [django.request:231] Internal Server Error: /processoren/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/rik/A-Pc/Website/pcbuilder/views.py", line 122, in processoren
    if float(diestringnaam) < float(minPriceSliderValue):
UnicodeEncodeError: 'decimal' codec can't encode character u'\u20ac' in position 0: invalid decimal Unicode string
[28/Nov/2014 11:51:55] ERROR [django.request:231] Internal Server Error: /processoren/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/rik/A-Pc/Website/pcbuilder/views.py", line 122, in processoren
    if float(diestringnaam) < float(minPriceSliderValue):
UnicodeEncodeError: 'decimal' codec can't encode character u'\u20ac' in position 0: invalid decimal Unicode string
[28/Nov/2014 11:51:58] ERROR [django.request:231] Internal Server Error: /processoren/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/rik/A-Pc/Website/pcbuilder/views.py", line 122, in processoren
    if float(diestringnaam) < float(minPriceSliderValue):
UnicodeEncodeError: 'decimal' codec can't encode character u'\u20ac' in position 0: invalid decimal Unicode string
[28/Nov/2014 11:51:58] ERROR [django.request:231] Internal Server Error: /processoren/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/rik/A-Pc/Website/pcbuilder/views.py", line 122, in processoren
    if float(diestringnaam) < float(minPriceSliderValue):
UnicodeEncodeError: 'decimal' codec can't encode character u'\u20ac' in position 0: invalid decimal Unicode string
[28/Nov/2014 11:51:58] ERROR [django.request:231] Internal Server Error: /processoren/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/rik/A-Pc/Website/pcbuilder/views.py", line 122, in processoren
    if float(diestringnaam) < float(minPriceSliderValue):
UnicodeEncodeError: 'decimal' codec can't encode character u'\u20ac' in position 0: invalid decimal Unicode string
[28/Nov/2014 11:51:59] ERROR [django.request:231] Internal Server Error: /processoren/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/rik/A-Pc/Website/pcbuilder/views.py", line 122, in processoren
    if float(diestringnaam) < float(minPriceSliderValue):
UnicodeEncodeError: 'decimal' codec can't encode character u'\u20ac' in position 0: invalid decimal Unicode string
[28/Nov/2014 11:51:59] ERROR [django.request:231] Internal Server Error: /processoren/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/rik/A-Pc/Website/pcbuilder/views.py", line 122, in processoren
    if float(diestringnaam) < float(minPriceSliderValue):
UnicodeEncodeError: 'decimal' codec can't encode character u'\u20ac' in position 0: invalid decimal Unicode string
[28/Nov/2014 11:52:31] ERROR [django.request:231] Internal Server Error: /processoren/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/rik/A-Pc/Website/pcbuilder/views.py", line 122, in processoren
    if float(diestringnaam) < float(minPriceSliderValue):
UnicodeEncodeError: 'decimal' codec can't encode character u'\u20ac' in position 0: invalid decimal Unicode string
[30/Nov/2014 11:26:58] WARNING [django.request:143] Not Found: /favicon.ico
[30/Nov/2014 11:27:02] ERROR [django.request:231] Internal Server Error: /processoren/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/rik/A-Pc/Website/pcbuilder/views.py", line 122, in processoren
    if float(diestringnaam) < float(minPriceSliderValue):
UnicodeEncodeError: 'decimal' codec can't encode character u'\u20ac' in position 0: invalid decimal Unicode string
